app/src/main/res/layout/fragment_fmt.xml

```diff
@@ -47,18 +46,34 @@
         <com.google.android.material.button.MaterialButton
                 android:id="@+id/reset_button"
                 style="@style/Widget.MaterialComponents.Button.OutlinedButton"
-                android:layout_width="match_parent"
+                android:layout_width="0dp"
                 android:layout_height="wrap_content"
                 android:layout_marginStart="8dp"
                 android:layout_marginEnd="8dp"
                 android:text="@string/fmt_timer_reset"
                 android:visibility="invisible"
-                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintEnd_toStartOf="@id/edit_time_button"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintBottom_toTopOf="@id/timer_view2"
                 tools:visibility="visible"
                 android:layout_marginBottom="8dp" />
 
+        <com.google.android.material.button.MaterialButton
```

Comment:
Can you remove the button here and add it to the action bar? I did that in the buzzers fragment.